www/src/templates/docs.js

```diff
@@ -30,13 +31,25 @@ const sectionList = sectionListA
     }),
   );
 
-const Docs = ({data, location}) => (
-  <MarkdownPage
-    location={location}
-    markdownRemark={data.markdownRemark}
-    sectionList={sectionList}
-  />
-);
+const Docs = ({data, location}) => {
+  console.log('location is ', location);
+  if (location.pathname === '/docs/installation.html') {
```

Comment:
This _works_ but I think we'd be better off doing this check in `gatsby-node` like we do [for the Home page](https://github.com/facebook/react/blob/gatsby/www/gatsby-node.js#L63-L70).

Reply:
Cool - will look into that.